src/views/Shift.vue

```diff
@@ -132,13 +154,12 @@ export default {
   mounted () {
     this.nextOrder();
     salesService.getShift(parseInt(this.shiftId)).then((shift) => {this.shift = shift; this.recalculateProgress()});
-    this.fetchShiftInterval = setInterval(this.fetchShiftUpdates, 20000);
-    this.progressInterval = setInterval(this.recalculateProgress, 5000);
+    this.fetchShiftInterval = setInterval(this.fetchShiftUpdates, this.SHIFT_REFRESH_RATE);
```

Comment:
This should probably also be a repeating setTimeout call, since it can still result in stacking request upon request if it takes really long. Even though the interval is really long.

Reply:
I figured it shouldnt be a problem here

Reply:
Do you want me to change it?

Reply:
I wouldn't have commented it otherwise :p